libs/clist/clist_test.go

```diff
@@ -17,18 +17,18 @@ func TestPanicOnMaxLength(t *testing.T) {
 
 	l := newWithMax(maxLength)
 	for i := 0; i < maxLength; i++ {
-		l.PushBack(1)
+		l.PushBack(1, 0)
```

Comment:
too many places calling `PushBack` have to be updated. how about keep `PushBack` unchanged, while adding `PushBackWithPriority` method, while the former will insert default priority 0?